src/sentry/lang/javascript/processor.py

```diff
@@ -275,27 +281,30 @@ def fetch_release_file(filename, release, dist=None):
                 rf for ident in filename_idents for rf in possible_files if rf.ident == ident
             )
 
+        logger.debug(
+            "Found release artifact %r (id=%s, release_id=%s)", filename, releasefile.id, release.id
+        )
```

Comment:
Moved this up to a section of the implementation where it's more logically coupled.